backend/src/migrations/20240216181434_add_columns_to_user_details_table.ts

```diff
@@ -27,7 +27,7 @@ async function up(knex: Knex): Promise<void> {
     await knex.schema.alterTable(TABLE_NAME, (table) => {
         table.text(ColumnName.AVATAR_URL).nullable();
         table.string(ColumnName.USERNAME).unique().nullable();
-        table.date(ColumnName.DATE_OF_BIRTH).nullable();
+        table.dateTime(ColumnName.DATE_OF_BIRTH).nullable();
```

Comment:
Date will be enough, also don't change existing migrations, it's already deployed to stage